<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deletePage</class>
 <widget class="QWidget" name="deletePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>41</width>
     <height>91</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>1</number>
   </property>
   <property name="intValue" stdset="0">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>41</width>
     <height>91</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>1</number>
   </property>
   <property name="intValue" stdset="0">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>41</width>
     <height>91</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>1</number>
   </property>
   <property name="intValue" stdset="0">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>250</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入要抛弃的精灵号码：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
